CHANGELOG.adoc

```diff
@@ -53,6 +53,7 @@ Rename the class `SemanticCheckerFactory` to `SemanticRunnableFactory` to reflec
 - https://github.com/eclipse-syson/syson/issues/771[#771] [diagrams] Allow the drop of elements on empty diagram nodes.
 - https://github.com/eclipse-syson/syson/issues/775[#775] [syson] Exclude `src/test/resources` from checkstyle.
 This reduces the time required to build SysON, especially when using Spring Tool Suite.
+- https://github.com/eclipse-syson/syson/issues/779[#779] [diagrams] Add a tool in Part usage to create a subsetting edge and the Part usage that is subset.
```

Comment:
"Part usage" -> "`PartUsage`" (note the "`")

Reply:
and the Part usage -> and a `PartUsage`

Reply:
subsetting -> `Subsetting`

Reply:
tool in -> tool on

Reply:
Could you also add the name of this new tool in this message please?

Reply:
Done